File: ex-13/ex-13_java/Principal.java
import java.util.Scanner;

public class Principal
{
    private Vetor v1, v2;

    //Construtora
    public Principal()
    {
        getEntradas();
    }

    public void executar()
    {
        Vetor soma = v1.soma(v2);
        Vetor mult1 = v1.multiplica(4);
        Vetor mult2 = v2.multiplica(2);
        System.out.println("Soma dos vetores: <" + soma.getDx() + "," + soma.getDy() + ">");
        System.out.println("Multiplicação vetor 1: <" + mult1.getDx() + "," + mult1.getDy() + ">");
        System.out.println("Multiplicação vetor 2: <" + mult2.getDx() + "," + mult2.getDy() + ">");
    }

    public void getEntradas()
    {
        int x, y;
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digita os valores de x e y do primeiro vetor <x> <y>:");
        x = scanner.nextInt();
        y = scanner.nextInt();
        System.out.println();

        v1 = new Vetor(x, y);

        System.out.print("Digite os valores de x e y do segundo vetor <x> <y>: ");
        x = scanner.nextInt();
        y = scanner.nextInt();
        System.out.println();

        v2 = new Vetor(x, y);
    }

    public static void main(String[] args)
    {
        Principal principal = new Principal();

        principal.executar();
    }
}

File: ex-13/ex-13_python/Principal.py
from Vetor import Vetor


class Principal:
    # Construtora
    def __init__(self):
        self.v1 = Vetor()
        self.v2 = Vetor()
        self.getEntradas()

    def executar(self):
        soma = self.v1.soma(self.v2)
        mult1 = self.v1.multiplica(4)
        mult2 = self.v2.multiplica(2)
        print("Soma dos vetores: <" + str(soma.getDx()) + "," + str(soma.getDy()) + ">")
        print("Multiplicação vetor 1: <" + str(mult1.getDx()) + "," + str(mult1.getDy()) + ">")
        print("Multiplicação vetor 2: <" + str(mult2.getDx()) + "," + str(mult2.getDy()) + ">")

    def getEntradas(self):
        print("Digita os valores de x e y do primeiro vetor <x> <y>:", end="")
        x, y = map(int, input().split())
        print()

        self.v1 = Vetor(x, y)

        print("Digite os valores de x e y do segundo vetor <x> <y>: ", end="")
        x, y = map(int, input().split())
        print()

        self.v2 = Vetor(x, y)
